gnmi_server/server.go

```diff
@@ -376,6 +382,18 @@ func (s *Server) Get(ctx context.Context, req *gnmipb.GetRequest) (*gnmipb.GetRe
 	return &gnmipb.GetResponse{Notification: notifications}, nil
 }
 
+// SaveOnSetEnabled saves configuration to a file
+func SaveOnSetEnabled() {
```

Comment:
Do you need this function to be exposed to other packages? Shouldn't this be a local only function?

Reply:
Done